fw-child/resources/scss/_pages.scss

```diff
@@ -56,9 +66,10 @@
   --swiper-pagination-bullet-horizontal-gap: 0.75rem;
   --swiper-pagination-bullet-inactive-color: #{$blue-200};
   --swiper-pagination-color: #{$white};
+  --swiper-pagination-bottom: -5rem;
```

Comment:
This looks bad on iPhone 5 (320x568). You can use the existing custom breakpoint 'xsm' which is at 400px if needed.

Reply:
Sure, I applied after `xsm` to avoid that.

Reply:
Sure, I applied after xsm to avoid that.